recipes/peewee/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{%set name = "peewee" %}
+{%set version = "2.8.1" %}
+{%set hash_type = "md5" %}
+{%set hash_val = "542f5c3e7792c14da6c5bc385f90ca77" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+
+  number: 0
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - cython >=0.22.1
+
+  run:
+    - python
+
+test:
+  imports:
+    - playhouse
```

Comment:
@dharhas I am not a `peewee` user so I am not sure if that is the only import worth testing. Can you check that?

PS: It seems we have a successful build :tada:  I will merge this as soon.